cmd/notification.go

```diff
@@ -1054,6 +1054,29 @@ func (sys *NotificationSys) CPULoadInfo() []ServerCPULoadInfo {
 	return reply
 }
 
+// CPUInfo - CPU Hardware info
+func (sys *NotificationSys) CPUInfo() []madmin.ServerCPUHardwareInfo {
+	reply := make([]madmin.ServerCPUHardwareInfo, len(sys.peerClients))
+	var wg sync.WaitGroup
+	for i, client := range sys.peerClients {
+		if client == nil {
+			continue
```

Comment:
Are we just going to leave `reply[i]` empty/zero values?

Reply:
Yes, if client is nil reply[i] is empty value. I will filter these response and give the o/p in `mc`